client/benchmark/src/main/java/org/elasticsearch/client/benchmark/ops/search/SearchBenchmarkTask.java

```diff
@@ -57,22 +57,17 @@ public void run() throws Exception {
 
             int waitTime = (int) Math.floor(MICROS_PER_SEC / targetThroughput - (stop - start) / NANOS_PER_MICRO);
             if (waitTime > 0) {
-                // Thread.sleep() time is not very accurate (it's most of the time around 1 - 2 ms off)
-                // so we rather busy spin for the last few microseconds. Still not entirely accurate but way closer
                 waitMicros(waitTime);
```

Comment:
you mentioned in the description that "We significantly reduce measurement skew at low target throughput rates". How is that achieved? Is that below here?